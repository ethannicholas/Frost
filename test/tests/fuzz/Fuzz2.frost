method main() {
    var i := 0
    do {
        Console.printLine(i)
        i += 1
    }
    while i.< 10
}
