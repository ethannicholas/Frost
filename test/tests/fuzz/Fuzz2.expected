Fuzz2.frost:7:12: error: reference to method '<' cannot be used as type 'frost.core.Bit'. None of the following methods match this type:
    function frost.core.Int.<(other:frost.core.Int8):frost.core.Bit:639:5 (cannot be an instance of frost.core.Bit)
    function frost.core.Int.<(other:frost.core.Int16):frost.core.Bit:645:5 (cannot be an instance of frost.core.Bit)
    function frost.core.Int.<(other:frost.core.Int32):frost.core.Bit:651:5 (cannot be an instance of frost.core.Bit)
    function frost.core.Int.<(other:frost.core.Int64):frost.core.Bit:657:5 (cannot be an instance of frost.core.Bit)
    function frost.core.Int.<(other:frost.core.Int):frost.core.Bit:664:5 (cannot be an instance of frost.core.Bit)
    function frost.core.Int.<(other:frost.core.UInt8):frost.core.Bit:670:5 (cannot be an instance of frost.core.Bit)
    function frost.core.Int.<(other:frost.core.UInt16):frost.core.Bit:676:5 (cannot be an instance of frost.core.Bit)
    function frost.core.Int.<(other:frost.core.UInt32):frost.core.Bit:682:5 (cannot be an instance of frost.core.Bit)
    function frost.core.Int.<(other:frost.core.UInt):frost.core.Bit:688:5 (cannot be an instance of frost.core.Bit)
Fuzz2.frost:7:15: error: not a statement
2 errors
